src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -0,0 +1,291 @@
+package tech.cryptonomic.conseil.tezos
+
+import tech.cryptonomic.conseil.tezos.TezosTypes._
+import tech.cryptonomic.conseil.model.Model
+import tech.cryptonomic.conseil.tezos.FeeOperations._
+import tech.cryptonomic.conseil.util.Conversion
+import tech.cryptonomic.conseil.util.Conversion.Id
+
+object DatabaseConversions {
```

Comment:
I think this library could help us avoid some of boilerplate code in this object: https://github.com/scalalandio/chimney

What do you think?

Reply:
Thanks, I know about the library, of course 😄 
I still didn't use any additional resource to avoid obfuscating the code behind new apis.
This is already introducing a new concept to be handled for anyone not familiar with typeclasses, so I tried to stick to manual and explicit conversion, as long as it's manageable. (part of the code was simply moved from another existing class to here).

To sum it up, I'll have a better look at Chimney to evaluate if it provides substantial help without adding extra complexity

Reply:
I might consider this, yet in a separate task and PR, this is getting too chubby for me